viewing_party/party.py

```diff
@@ -0,0 +1,181 @@
+#  WAVE 1 #
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        movie_dict = {}
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+    else: 
+        return None
+
+def add_to_watched(user_data, movie):
+    if "watched" in user_data.keys():
+        user_data["watched"].append(movie)
+    else:
+        user_data["watched"] = [movie]
+
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    
+    return user_data
+
+def watch_movie(user_data, title):
+
+    count = 0
+    
+    for item in user_data["watchlist"]:
+        count += 1
+        if item["title"] == title:
+            watched_movie = user_data["watchlist"][count-1]
+            user_data["watchlist"].remove(watched_movie)
```

Comment:
even though remove works great here, be careful using it in a for loop. It can cause side effects. Here is a link that explains it more with examples https://thispointer.com/python-remove-elements-from-a-list-while-iterating/